### PR TITLE
feat: added ip-restriction form

### PR DESCRIPTION
**Pull request by iamayushdas**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

Please update this section with detailed description.

**Related issues**

fix/resolve #0001

**Checklist:**

- [ ] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [ ] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Comment by iamayushdas** [comment]
cc @juzhiyuan

File: web/src/components/Plugin/UI/ip-restriction.tsx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+                              id: 'component.global.pleaseEnter',
+                            })} whitelist`,
+                          },
+                        ]}
+                      >
+                        <Input />
+                      </Form.Item>
+                    </Col>
+                    <Col style={{ ...removeBtnStyle, marginLeft: -10 }}>
+                      {fields.length > 1 ? (
```
Comment (Baoyuantop) [inline web/src/components/Plugin/UI/ip-restriction.tsx:95]: may better with `&&`?

File: web/src/components/Plugin/UI/plugin.tsx [hunk 2]
```diff
@@ -27,6 +27,11 @@ import ProxyMirror from './proxy-mirror';
 import LimitConn from './limit-conn';
 import RefererRestriction from './referer-restriction';
 import Cors from './cors';
+<<<<<<< HEAD
+=======
+import RequestId from './request-id';
+import IpRestriction from './ip-restriction';
+>>>>>>> 18ade18 (feat: ip-restriction added to plugin list)
```
Comment (Baoyuantop) [inline web/src/components/Plugin/UI/plugin.tsx:34]: Are there conflicting files that have not been dealt with yet?

Comment (iamayushdas) [reply web/src/components/Plugin/UI/plugin.tsx:34]: Yes this PR is in a draft mode right now, it may have many errors

**Comment by liuxiran** [comment]
Hi @iamayushdas if there are any problems please let me know, thanks

**Pull request closed without merging**